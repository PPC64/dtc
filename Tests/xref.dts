/dts-v1/;

/ {
        foo {
                bar {
                        fish = <1>;
                };
        };
};

/ {
        baz = <&{/foo/bar}>;
};
