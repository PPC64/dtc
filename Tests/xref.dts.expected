/dts-v1/;

/  {

	baz = <0x1>;
	foo {

		bar {

			fish = <0x1>;
			phandle = <0x1>;
		};
	};
};
